.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+          cache-all-crates: true
+          cache-workspace-crates: true
```

actionlint (1.7.7)

input "cache-workspace-crates" is not defined in action "Swatinem/rust-cache@v2". available inputs are "cache-all-crates", "cache-bin", "cache-directories", "cache-on-failure", "cache-provider", "cache-targets", "env-vars", "key", "lookup-only", "prefix-key", "save-if", "shared-key", "workspaces"

(action)